docs/develop/documentation.rst

```diff
@@ -6,16 +6,27 @@ The documentation for ``datatable`` project is written entirely in the
 ReStructured Text (RST) format and rendered using the Sphinx engine. These
 technologies are standard for Python.
 
-The basic workflow for developing documentation, after
-:ref:`setting up a local datatable repository <local-setup>`, is to go into
-the ``docs/`` directory and run
+The source RST files for the documentation live in the ``/docs`` directory
+of the project. With every pull request they are sent to ``readthedocs.io``
```

Comment:
We could make ``readthedocs.io`` here to be a hyperlink.

Reply:
Well, I was thinking that, since the users are already reading this documentation on `readthedocs.io`, they might be able to figure it out. :)